AmethystRuntime/src/AmethystRuntime.h

```diff
@@ -65,9 +63,14 @@ class AmethystRuntime {
     }
 
     static Amethyst::PatchManager* getPatchManager() 
-	{
-		return &AmethystRuntime::getInstance()->mAmethystContext.mPatchManager;
-	}
+    {
+        return &AmethystRuntime::getInstance()->mAmethystContext.mPatchManager;
+    }
+
+    static std::vector<Mod> getMods()
```

Comment:
The getter should return a pointer to the vector
```
static std::vector<Mod>* getMods() 
{
    &return AmethystRuntime::getInstance()->mAmethystContext.mMods;
}

Reply:
After this should be good to merge 👍

Reply:
done